### ISSUE TITLE
How to replace ${HOME} with Docker volume when starting Selenoid using cm in Docker

### ISSUE THREAD

**Issue by firyx** [aerokube/cm#180]
A way to start Selenoid using cm in Docker is:

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock  \
    -v ${HOME}:/root                                            \
    -e OVERRIDE_HOME=${HOME}                                    \
    aerokube/cm:latest-release selenoid start --vnc --tmpfs 128
```

But how can I replace ${HOME} with a Docker volume?

I have tried e.g. (where aerokube is a Docker volume name):

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
  -v aerokube:/root \
  aerokube/cm:latest-release selenoid start --vnc --tmpfs 128
```

In volume I can see `.aerokube` directory created,
cm logs finish with:

```
> Configuration saved to /root/.aerokube/selenoid/browsers.json
> Starting Selenoid...
> Successfully started Selenoid
```

But the container keeps restarting with logs:

```
... [-] [INIT] [Loading configuration files...]
... [-] [INIT] [/usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory]
```

I have same issue, when I try:

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
  -v aerokube:/root \
  -e OVERRIDE_HOME=/root \
  aerokube/cm:latest-release selenoid start --vnc --tmpfs 128
```

Is there a way to have stored `.aerokube` in Docker volume?

**Comment by vania-pooh** [comment]
@firyx there is a `cm` flag called `--config-dir` where you can override you configuration dir.
```
  -c, --config-dir string         directory to save files (default "/Users/vania-pooh/.aerokube/selenoid")
```
All the flags can be listed like the following:
```
./cm selenoid start --help
```

**Comment by firyx** [comment]
@vania-pooh

I have deleted all aerokube and selenoid images and tried (aerokube is a Docker volume):

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
  -v aerokube:/root \
  aerokube/cm:latest-release selenoid start --vnc --tmpfs 128 --config-dir /root/.aerokube/selenoid
```

Console output ends with:

```
> Configuration saved to /root/.aerokube/selenoid/browsers.json
> Starting Selenoid...
> Successfully started Selenoid 
```

In the aerokube volume are created folders and browsers.json:

`.aerokube/selenoid/browsers.json`

But the container still keeps restarting and from the logs taken by `docker logs --tail 50 --follow --timestamps selenoid`

I still can see:

```
...
... [-] [INIT] [Loading configuration files...]
... [-] [INIT] [/usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory]
... [-] [INIT] [Loading configuration files...]
... [-] [INIT] [/usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory]
```

When I tried instead of `--config-dir /root/.aerokube/selenoid` this `--config-dir /root/abc`,
then in aerokube volume the `abc/browsers.json` is created, but the container behaves the same way (restarts with the same logs)

**Comment by vania-pooh** [comment]
@firyx all these commands are too complicated. How about just trying `cm` binary instead of container? This is the case when using `cm` in Docker container complicates things.

**Comment by firyx** [comment]
@vania-pooh 
The reason, why we wanted to have it in docker and docker volume was, that we expect to move the Selenoid grid from current server to more powerful server(s) later. If we can move (export & import) Docker volumes and containers, we do not need to recreate folders, permissions, scripts etc. on the new server.

We made it working without cm, using just selenoid in docker.

Moreover I have noticed, that in cm in docker --no-download flag doesn't seem to work (in the logs it checked images in the hub).

Anyway,
Selenoid seems to be a great project!!!
Many thanks!!!
